### PR TITLE
Do not apply mandatory-files policy to openjdk sources

### PR DESCRIPTION
**Pull request by d3r3kk**
Our workflow requires that we mirror the upstream OpenJDK sources from their `master` branch to our `main` branch without any divergence. We then apply open-source Microsoft driven patches to a release branch, based off that mirrored `main` to build our binaries with.

We'd like to avoid diverging our `main` branch at all.

### CODE REVIEW

**Review by joe-braley — COMMENTED** [review]
Looks fine though I do have one question.

**Review by jeffwilcox — CHANGES_REQUESTED** [review]
this is blocked by an underlying deployment that is pending on the policy system to evaluate this correctly

File: policies/mandatory-files.yml [hunk 1]
```diff
@@ -5,6 +5,12 @@ description: this policy will ensure the presence of important files in Microsof
 # filters
 resource: repository
 where:
+- |
+  !repository.name.equals("openjdk-jdk", StringComparison.InvariantCultureIgnoreCase)
+  ||!repository.name.equals("openjdk-jdk11", StringComparison.InvariantCultureIgnoreCase)
```
[the remark this answers is not in the document]
Comment (d3r3kk) [reply policies/mandatory-files.yml]: Yes, caught that just after my submission - looks right now (I think).

**Review by DragosDanielBoia — APPROVED** [review]

**Review by jeffwilcox — APPROVED** [review]

File: policies/mandatory-files.yml [hunk 2]
```diff
@@ -5,6 +5,12 @@ description: this policy will ensure the presence of important files in Microsof
 # filters
 resource: repository
 where:
+- |
+  !repository.name.equals("openjdk-jdk", StringComparison.InvariantCultureIgnoreCase)
+  ||!repository.name.equals("openjdk-jdk11u", StringComparison.InvariantCultureIgnoreCase)
```
[the remark this answers is not in the document]
Comment (d3r3kk) [reply policies/mandatory-files.yml]: Thanks for taking care of this for us!

**Pull request merged**